### ISSUE TITLE
Need instructions to run your project

### ISSUE THREAD

**Issue by SpaceQuester** [SuperFluffy/oscillators#1]
Hi, can you, please,  add the instructions and recommendations how to run your program? What is OS, compiler, commands for run, etc?

Thank you!

**Comment by SuperFluffy** [comment]
Hey there. I have to admit that I did not actually continue working on this code, and have shifted to a different language (the code is not released).

I have just checked the Python code, and you should be able to run it with a current version of Python (3.6 in my case).

Sorry about that. :( The implementation of the Kuramoto model, for example, is actually quite straight forward.

**Comment by SpaceQuester** [comment]
Okey. Kuramoto oscillators network simulation code is only - "kuramoto.py" file? And it is run under Python 3.6. Is it right?
What is the files: "kuramoto.cc" and "kuramoto.hh"?

I am interesting only in Kuramoto network realization.

**Comment by SuperFluffy** [comment]
Yes, that's the one. You also need to have the `numpy` and `matplotlib` libraries (see the `import` statements at the beginning of the file).

If you are interested in networks, you need to play with the `coupling` matrix, which is responsible for “connecting” the various oscillators, see [this line](https://github.com/SuperFluffy/oscillators/blob/master/kuramoto.py#L51).

 The other files pertain to an implementation in C++, which I am not even sure I completed.
